addons/sms/models/ir_actions_server.py

```diff
@@ -49,7 +49,7 @@ def _check_model_coherency(self):
         super()._check_model_coherency()
         for action in self:
             if action.state == 'sms' and (action.model_id.transient or not action.model_id.is_mail_thread):
-                raise ValidationError(_("Sending SMS can only be done on a mail.thread or a transient model"))
+                raise ValidationError(_("Sending SMS can only be done on a mail.thread or not a transient model"))
```

Comment:
Indeed ... but it is an and between the two, hence
```suggestion
                raise ValidationError(_("Sending SMS can only be done on a not transient mail.thread"))
```

Reply:
Sure, as you wish sir <3